zbm-builder.sh

```diff
@@ -102,7 +124,7 @@ fi
 
 # If no config is specified, use in-tree default
 if ! [ -r ./config.yaml ]; then
```

Comment:
To support a custom build directory, this should be
```suggestion
if ! [ -r "${build_mnt}/config.yaml" ]; then
```

Reply:
:+1: